pkg/ginkgo-reporters/polarion_reporter.go

```diff
@@ -215,8 +216,18 @@ func (reporter *PolarionReporter) SpecSuiteDidEnd(summary *types.SuiteSummary) {
 	reporter.Suite.Time = summary.RunTime.Seconds()
 	reporter.Suite.Failures = summary.NumberOfFailedSpecs
 
+	// include parallel node number in output file name in case of ginkgo parallel runs
+	outputFile := reporter.Filename
+	if reporter.ParallelNode > 0 {
+		if strings.Contains(reporter.Filename, "%d") {
```

Comment:
In kubevirt we have this:

```
	if config.GinkgoConfig.ParallelTotal > 1 {
		artifactsPath = path.Join(artifactsPath, strconv.Itoa(config.GinkgoConfig.ParallelNode))
		junitOutput = path.Join(flags.ArtifactsDir, fmt.Sprintf("partial.junit.functest.%d.xml", config.GinkgoConfig.ParallelNode))
	}
```

Note that `config` comes from `github.com/onsi/ginkgo/config`.